src/app/(tabs)/map.tsx

```diff
@@ -109,6 +111,25 @@ export default function Screen(): JSX.Element {
 }
 
 export const MapScreen = (): JSX.Element => {
+    const [userData, setUserData] = useState<PersDataDetails | null>(null)
+
+    useEffect(() => {
+        async function load(): Promise<void> {
+            try {
+                const response = await API.getPersonalData()
```

Comment:
We should avoid making an persData API request every time we open the map. Instead, we could extend the UserKind context hook and store the faculty there.